### DISCUSSION TITLE
[Question][Hw1] Add new variable in homework

### DISCUSSION THREAD

**Discussion by reinattwijaya** [prosyslab-classroom/cs348-information-security#663] [Q&A]
Name: Reinatt Hansel Wijaya

Is it okay if we add new variable (add let or let rec) in the homework?

**Comment by sujin0529** [comment]
Hi,

Yes, you can add a new variable to the homework if you need a new variable.

Thanks.

**Comment by KihongHeo** [comment]
@reinattwijaya let me close.
